src/Contracts/IndexesResults.php

```diff
@@ -20,7 +20,7 @@ public function __construct(array $params)
     }
 
     /**
-     * @return array<int, array>
+     * @return array<int, Indexes>
```

Comment:
Php stan still doesn't seem happy with this, it is normal?

Reply:
Because the `Indexes` import is missing :)

Reply:
Also not sure why introducing it here if I've already made a PR for this fix? :D